deb2rpm.pl

```diff
@@ -102,6 +102,9 @@ sub write_spec {
     if (-e "debian/$package.install") {
       print $out "install_to_buildroot < %{rootdir}/debian/$package.install\n";
     }
+    if (-e "debian/$package.links") {
+      print $out "install_links_in_buildroot < %{rootdir}/debian/$package.links\n";
```

Comment:
What's the logic behind putting the `rsyslogd.conf` file in the packaging environment, rather than in the filesystem tree? Is it